### ISSUE TITLE
IPv6 Support

### ISSUE THREAD

**Issue by majst01** [metal-stack/metal-networker#41]
nftables rules are note ipv6 ready:

```
{"level":"panic","ts":1611306775.7591274,"caller":"netconf/configurator.go:249","msg":"exit status 1: /etc/metal/networker/rules.v4_861797323:45:39-44: Error: Could not resolve hostname: Address family for hostname not supported\n        oifname \"vlan104008\" ip saddr 2002::/64 counter masquerade comment \"snat (networkid: ipv6)\"\n                                      ^^^^^^\n","stacktrace":"github.com/metal-stack/metal-networker/internal/netconf.mustApply\n\tgithub.com/metal-stack/metal-networker/internal/netconf/configurator.go:249\ngithub.com/metal-stack/metal-networker/internal/netconf.applyAndCleanUp\n\tgithub.com/metal-stack/metal-networker/internal/netconf/configurator.go:223\ngithub.com/metal-stack/metal-networker/internal/netconf.FirewallConfigurator.Configure\n\tgithub.com/metal-stack/metal-networker/internal/netconf/configurator.go:101\ngithub.com/metal-stack/metal-networker/cmd.configure\n\tgithub.com/metal-stack/metal-networker/cmd/root.go:96\ngithub.com/metal-stack/metal-networker/cmd.glob..func1\n\tgithub.com/metal-stack/metal-networker/cmd/firewall.go:24\ngithub.com/spf13/cobra.(*Command).execute\n\tgithub.com/spf13/cobra@v1.1.1/command.go:850\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\tgithub.com/spf13/cobra@v1.1.1/command.go:958\ngithub.com/spf13/cobra.(*Command).Execute\n\tgithub.com/spf13/cobra@v1.1.1/command.go:895\ngithub.com/metal-stack/metal-networker/cmd.Execute\n\tgithub.com/metal-stack/metal-networker/cmd/root.go:43\nmain.main\n\tgithub.com/metal-stack/metal-networker/main.go:6\nruntime.main\n\truntime/proc.go:204"}
panic: exit status 1: /etc/metal/networker/rules.v4_861797323:45:39-44: Error: Could not resolve hostname: Address family for hostname not supported
        oifname "vlan104008" ip saddr 2002::/64 counter masquerade comment "snat (networkid: ipv6)"
                                      ^^^^^^


goroutine 1 [running]:
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc0002ec180, 0x0, 0x0, 0x0)
        go.uber.org/zap@v1.16.0/zapcore/entry.go:234 +0x585
go.uber.org/zap.(*SugaredLogger).log(0xc0000114d0, 0xc00008cc04, 0x0, 0x0, 0xc0005695d8, 0x1, 0x1, 0x0, 0x0, 0x0)
        go.uber.org/zap@v1.16.0/sugar.go:234 +0xf6
go.uber.org/zap.(*SugaredLogger).Panic(...)
        go.uber.org/zap@v1.16.0/sugar.go:123
github.com/metal-stack/metal-networker/internal/netconf.mustApply(0xc00e60, 0xc0004e80f0, 0xc000318a80, 0x926, 0xc00008cc90, 0x27, 0xb2c4c7, 0x16)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:249 +0x1de
github.com/metal-stack/metal-networker/internal/netconf.applyAndCleanUp(0xc00e60, 0xc0004e80f0, 0xb26ae2, 0xc, 0xc00008cc90, 0x27, 0xb2c4c7, 0x16, 0x180)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:223 +0x1cb
github.com/metal-stack/metal-networker/internal/netconf.FirewallConfigurator.Configure(0xc0002a69b8, 0x2, 0x0, 0x0, 0x0, 0x0, 0xc000124600, 0x3, 0x3, 0xc00008cfc0, ...)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:101 +0x1b9
github.com/metal-stack/metal-networker/cmd.configure(0x3, 0xf69c40, 0x0, 0x0)
        github.com/metal-stack/metal-networker/cmd/root.go:96 +0x2b2
github.com/metal-stack/metal-networker/cmd.glob..func1(0xf69c40, 0xc0002a0d40, 0x0, 0x2, 0x0, 0x0)
        github.com/metal-stack/metal-networker/cmd/firewall.go:24 +0x34
github.com/spf13/cobra.(*Command).execute(0xf69c40, 0xc0002a0d20, 0x2, 0x2, 0xf69c40, 0xc0002a0d20)
        github.com/spf13/cobra@v1.1.1/command.go:850 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xf6a420, 0xc000011528, 0xc000374a20, 0xc000374a20)
        github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.1.1/command.go:895
github.com/metal-stack/metal-networker/cmd.Execute()
        github.com/metal-stack/metal-networker/cmd/root.go:43 +0xe5
```

**Comment by majst01** [comment]
So next:

```
{"level":"info","ts":1611507404.867368,"caller":"netconf/configurator.go:221","msg":"rendering hosts.tpl to /etc/hosts (mode: -rw-------)"}                                                                                                                                               
{"level":"info","ts":1611507404.8816912,"caller":"netconf/configurator.go:221","msg":"rendering hostname.tpl to /etc/hostname (mode: -rw-r--r--)"}                                                                                                                                        
{"level":"info","ts":1611507404.9009247,"caller":"netconf/configurator.go:221","msg":"rendering frr.firewall.tpl to /etc/frr/frr.conf (mode: -rw-------)"}                                                                                                                                
{"level":"info","ts":1611507404.918974,"caller":"netconf/frr.go:89","msg":"running 'vtysh --dryrun --inputfile /etc/metal/networker/frr_262413477' to validate changes.'"}                                                                                                                
{"level":"info","ts":1611507405.097335,"caller":"netconf/configurator.go:221","msg":"rendering rules.v4.tpl to /etc/nftables/rules.v4 (mode: -rw-------)"}                                                                                                                                
{"level":"info","ts":1611507405.1013398,"caller":"netconf/nftables.go:91","msg":"running 'nft --check --file /etc/metal/networker/rules.v4_136206272' to validate changes."}                                                                                                              
{"level":"info","ts":1611507405.1109192,"caller":"netconf/configurator.go:221","msg":"rendering rules.v6.tpl to /etc/nftables/rules.v6 (mode: -rw-------)"}                                                                                                                               
{"level":"info","ts":1611507405.1162035,"caller":"netconf/nftables.go:97","msg":"running 'nft --check --file /etc/metal/networker/rules.v6_694184991' to validate changes."}                                                                                                              
{"level":"warn","ts":1611507405.126014,"caller":"netconf/configurator.go:109","msg":"failed to configure Chrony: there is no network providing a default (0.0.0.0/0) route"}                                                                                                              
{"level":"info","ts":1611507405.1300104,"caller":"netconf/configurator.go:221","msg":"rendering droptailer.service.tpl to /etc/systemd/system/droptailer.service (mode: -rw-r--r--)"}                                                                                                     
{"level":"warn","ts":1611507405.1378279,"caller":"netconf/configurator.go:123","msg":"failed to deploy firewall-controller.service service : there is no network providing a default (0.0.0.0/0) route"}                                                                                  
{"level":"info","ts":1611507405.1419415,"caller":"netconf/configurator.go:221","msg":"rendering firewall_controller.service.tpl to /etc/systemd/system/firewall-controller.service (mode: -rw-r--r--)"}                                                                                   
panic: runtime error: invalid memory address or nil pointer dereference                                                                                                                                                                                                                   
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x93e6b9]

goroutine 1 [running]:
github.com/metal-stack/metal-networker/internal/netconf.mustApply(0x0, 0x0, 0xc00073a000, 0x1d4, 0xc0003d4980, 0x39, 0xc00008de30, 0x2f)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:246 +0xb9
github.com/metal-stack/metal-networker/internal/netconf.applyAndCleanUp(0x0, 0x0, 0xb37b15, 0x1f, 0xc0003d4980, 0x39, 0xc00008de30, 0x2f, 0x1a4)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:223 +0x1cb
github.com/metal-stack/metal-networker/internal/netconf.FirewallConfigurator.Configure(0xc0003b4080, 0x3, 0x0, 0x0, 0x0, 0x0, 0xc000126600, 0x3, 0x3, 0xc00008c150, ...)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:126 +0x60b
github.com/metal-stack/metal-networker/cmd.configure(0x3, 0xf6fd00, 0x0, 0x0)
        github.com/metal-stack/metal-networker/cmd/root.go:96 +0x2b2
github.com/metal-stack/metal-networker/cmd.glob..func1(0xf6fd00, 0xc00039c0e0, 0x0, 0x2, 0x0, 0x0)
        github.com/metal-stack/metal-networker/cmd/firewall.go:24 +0x34
github.com/spf13/cobra.(*Command).execute(0xf6fd00, 0xc00039c0c0, 0x2, 0x2, 0xf6fd00, 0xc00039c0c0)
        github.com/spf13/cobra@v1.1.1/command.go:850 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xf704e0, 0xc000010010, 0x0, 0xc000356a80)
        github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.1.1/command.go:895
github.com/metal-stack/metal-networker/cmd.Execute()
        github.com/metal-stack/metal-networker/cmd/root.go:43 +0xe5
main.main()
        github.com/metal-stack/metal-networker/main.go:6 +0x25
exit status 2
running install.sh in chroot failed
github.com/metal-stack/metal-hammer/cmd.(*Hammer).install
        /work/cmd/install.go:125
github.com/metal-stack/metal-hammer/cmd.(*Hammer).Install
        /work/cmd/install.go:78
github.com/metal-stack/metal-hammer/cmd.(*Hammer).installImage
        /work/cmd/root.go:251
github.com/metal-stack/metal-hammer/cmd.Run
        /work/cmd/root.go:244
main.main
        /work/main.go:74
runtime.main
        /usr/local/go/src/runtime/proc.go:204
runtime.goexit
        /usr/local/go/src/runtime/asm_amd64.s:1374
install
github.com/metal-stack/metal-hammer/cmd.(*Hammer).installImage
        /work/cmd/root.go:255
github.com/metal-stack/metal-hammer/cmd.Run
        /work/cmd/root.go:244
main.main
        /work/main.go:74
runtime.main


```

**Comment by majst01** [comment]
Debian firewall fails with:

```
{"level":"info","ts":1610973925.3848753,"caller":"netconf/configurator.go:221","msg":"rendering rules.v4.tpl to /etc/nftables/rules.v4 (mode: -rw-------)"}                                                                                                                               
{"level":"info","ts":1610973925.3868678,"caller":"netconf/nftables.go:91","msg":"running 'nft --check --file /etc/metal/networker/rules.v4_877005929' to validate changes."}                                                                                                              
{"level":"panic","ts":1610973925.3963466,"caller":"netconf/configurator.go:249","msg":"exit status 1: /etc/metal/networker/rules.v4_877005929:45:60-69: Error: NAT is only supported for IPv4/IPv6\n        oifname \"vlan104009\" ip saddr 10.0.16.0/22 counter masquerade comment \"snat
 (networkid: internet-vagrant-lab)\"\n                                                           ^^^^^^^^^^\n","stacktrace":"github.com/metal-stack/metal-networker/internal/netconf.mustApply\n\tgithub.com/metal-stack/metal-networker/internal/netconf/configurator.go:249\ngithub.com/
metal-stack/metal-networker/internal/netconf.applyAndCleanUp\n\tgithub.com/metal-stack/metal-networker/internal/netconf/configurator.go:223\ngithub.com/metal-stack/metal-networker/internal/netconf.FirewallConfigurator.Configure\n\tgithub.com/metal-stack/metal-networker/internal/netconf/configurator.go:101\ngithub.com/metal-stack/metal-networker/cmd.configure\n\tgithub.com/metal-stack/metal-networker/cmd/root.go:96\ngithub.com/metal-stack/metal-networker/cmd.glob..func1\n\tgithub.com/metal-stack/metal-networker/cmd/firewall.go:24\ngithub.com/spf13/cobra.(*Command).execute\n\tgithub.com/spf13/cobra@v1.1.1/command.go:850\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\tgithub.com/spf13/cobra@v1.1.1/command.go:958\ngithub.com/spf13/cobra.(*Command).Execute\n\tgithub.com/spf13/cobra@v1.1.1/command.go:895\ngithub.com/metal-stack/metal-networker/cmd.Execute\n\tgithub.com/metal-stack/metal-networker/cmd/root.go:43\nmain.main\n\tgithub.com/metal-stack/metal-networker/main.go:6\nruntime.main\n\truntime/proc.go:204"}
panic: exit status 1: /etc/metal/networker/rules.v4_877005929:45:60-69: Error: NAT is only supported for IPv4/IPv6
        oifname "vlan104009" ip saddr 10.0.16.0/22 counter masquerade comment "snat (networkid: internet-vagrant-lab)"
                                                           ^^^^^^^^^^


goroutine 1 [running]:
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc000292180, 0x0, 0x0, 0x0)
        go.uber.org/zap@v1.16.0/zapcore/entry.go:234 +0x585
go.uber.org/zap.(*SugaredLogger).log(0xc0000114d8, 0xc000028d04, 0x0, 0x0, 0xc00074d5d8, 0x1, 0x1, 0x0, 0x0, 0x0)
        go.uber.org/zap@v1.16.0/sugar.go:234 +0xf6
go.uber.org/zap.(*SugaredLogger).Panic(...)
        go.uber.org/zap@v1.16.0/sugar.go:123
github.com/metal-stack/metal-networker/internal/netconf.mustApply(0xc050c0, 0xc0002e3620, 0xc0002da000, 0x92a, 0xc000028d20, 0x27, 0xb2ffca, 0x16)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:249 +0x1de
github.com/metal-stack/metal-networker/internal/netconf.applyAndCleanUp(0xc050c0, 0xc0002e3620, 0xb2a587, 0xc, 0xc000028d20, 0x27, 0xb2ffca, 0x16, 0x180)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:223 +0x1cb
github.com/metal-stack/metal-networker/internal/netconf.FirewallConfigurator.Configure(0xc000248870, 0x7, 0x0, 0x0, 0x0, 0x0, 0xc000120600, 0x3, 0x3, 0x0, ...)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:101 +0x1b9
github.com/metal-stack/metal-networker/cmd.configure(0x3, 0xf6fd00, 0x0, 0x0)
        github.com/metal-stack/metal-networker/cmd/root.go:96 +0x2b2
github.com/metal-stack/metal-networker/cmd.glob..func1(0xf6fd00, 0xc0002420e0, 0x0, 0x2, 0x0, 0x0)
        github.com/metal-stack/metal-networker/cmd/firewall.go:24 +0x34
github.com/spf13/cobra.(*Command).execute(0xf6fd00, 0xc0002420c0, 0x2, 0x2, 0xf6fd00, 0xc0002420c0)
        github.com/spf13/cobra@v1.1.1/command.go:850 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xf704e0, 0xc000010010, 0x0, 0xc000370a20)
        github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.1.1/command.go:895

```

**Comment by majst01** [comment]
Next issue, machine does not come up with:

```
{"level":"info","ts":1612173315.0305495,"caller":"netconf/frr.go:89","msg":"running 'vtysh --dryrun --inputfile /etc/metal/networker/frr_820149745' to validate changes.'"}
{"level":"panic","ts":1612173315.2168367,"caller":"netconf/configurator.go:249","msg":"exit status 2: line 27: % Unknown command[24]:  bgp router-id 2002::3\n","stacktrace":"github.com/metal-stack/metal-networker/internal/netconf.mustApply\n\tgithub.com/metal-stack/metal-networker/
internal/netconf/configurator.go:249\ngithub.com/metal-stack/metal-networker/internal/netconf.applyAndCleanUp\n\tgithub.com/metal-stack/metal-networker/internal/netconf/configurator.go:223\ngithub.com/metal-stack/metal-networker/internal/netconf.applyCommonConfiguration\n\tgithub.c
om/metal-stack/metal-networker/internal/netconf/configurator.go:217\ngithub.com/metal-stack/metal-networker/internal/netconf.MachineConfigurator.Configure\n\tgithub.com/metal-stack/metal-networker/internal/netconf/configurator.go:90\ngithub.com/metal-stack/metal-networker/cmd.confi
gure\n\tgithub.com/metal-stack/metal-networker/cmd/root.go:96\ngithub.com/metal-stack/metal-networker/cmd.glob..func2\n\tgithub.com/metal-stack/metal-networker/cmd/machine.go:23\ngithub.com/spf13/cobra.(*Command).execute\n\tgithub.com/spf13/cobra@v1.1.1/command.go:850\ngithub.com/s
pf13/cobra.(*Command).ExecuteC\n\tgithub.com/spf13/cobra@v1.1.1/command.go:958\ngithub.com/spf13/cobra.(*Command).Execute\n\tgithub.com/spf13/cobra@v1.1.1/command.go:895\ngithub.com/metal-stack/metal-networker/cmd.Execute\n\tgithub.com/metal-stack/metal-networker/cmd/root.go:43\nma
in.main\n\tgithub.com/metal-stack/metal-networker/main.go:6\nruntime.main\n\truntime/proc.go:204"}
panic: exit status 2: line 27: % Unknown command[24]:  bgp router-id 2002::3


goroutine 1 [running]:
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc0003e8180, 0x0, 0x0, 0x0)
        go.uber.org/zap@v1.16.0/zapcore/entry.go:234 +0x585
go.uber.org/zap.(*SugaredLogger).log(0xc0000114d0, 0xc000403d04, 0x0, 0x0, 0xc0006b3600, 0x1, 0x1, 0x0, 0x0, 0x0)
        go.uber.org/zap@v1.16.0/sugar.go:234 +0xf6
go.uber.org/zap.(*SugaredLogger).Panic(...)
        go.uber.org/zap@v1.16.0/sugar.go:123
github.com/metal-stack/metal-networker/internal/netconf.mustApply(0xc00e60, 0xc0002e3950, 0xc0002b6000, 0x521, 0xc000403d10, 0x22, 0xb2911c, 0x11)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:249 +0x1de
github.com/metal-stack/metal-networker/internal/netconf.applyAndCleanUp(0xc00e60, 0xc0002e3950, 0xb283cc, 0xf, 0xc000403d10, 0x22, 0xb2911c, 0x11, 0x180)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:223 +0x1cb
github.com/metal-stack/metal-networker/internal/netconf.applyCommonConfiguration(0x4, 0xc0003b8080, 0x4, 0x0, 0x0, 0x0, 0x0, 0xc00025c300, 0x1, 0x1, ...)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:217 +0x37e
github.com/metal-stack/metal-networker/internal/netconf.MachineConfigurator.Configure(...)
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:90
github.com/metal-stack/metal-networker/cmd.configure(0x4, 0xf6a180, 0x0, 0x0)
        github.com/metal-stack/metal-networker/cmd/root.go:96 +0x2b2
github.com/metal-stack/metal-networker/cmd.glob..func2(0xf6a180, 0xc00039c0e0, 0x0, 0x2, 0x0, 0x0)
        github.com/metal-stack/metal-networker/cmd/machine.go:23 +0x34
github.com/spf13/cobra.(*Command).execute(0xf6a180, 0xc00039c0c0, 0x2, 0x2, 0xf6a180, 0xc00039c0c0)
        github.com/spf13/cobra@v1.1.1/command.go:850 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xf6a420, 0xc000010010, 0x0, 0xc000374a20)
        github.com/spf13/cobra@v1.1.1/command.go:958 +0x375

```

**Comment by majst01** [comment]
Next issue creating a firewall: 

```
{"level":"info","ts":1612346174.2936003,"caller":"netconf/frr.go:89","msg":"running 'vtysh --dryrun --inputfile /etc/metal/networker/frr_300936362' to validate changes.'"}                                                                                                               
{"level":"info","ts":1612346174.4936588,"caller":"netconf/configurator.go:221","msg":"rendering rules.v4.tpl to /etc/nftables/rules.v4 (mode: -rw-------)"}                                                                                                                               
{"level":"info","ts":1612346174.4990785,"caller":"netconf/nftables.go:91","msg":"running 'nft --check --file /etc/metal/networker/rules.v4_205597697' to validate changes."}                                                                                                              
{"level":"panic","ts":1612346174.5089548,"caller":"netconf/configurator.go:249","msg":"exit status 1: /etc/metal/networker/rules.v4_205597697:45:39-44: Error: Could not resolve hostname: Address family for hostname not supported\n        oifname \"vlan104009\" ip saddr 2002::/64 co
unter masquerade comment \"snat (networkid: internet-vagrant-lab)\"\n                                      ^^^^^^\n","stacktrace":"github.com/metal-stack/metal-networker/internal/netconf.mustApply\n\tgithub.com/metal-stack/metal-networker/internal/netconf/configurator.go:249\ngithu
b.com/metal-stack/metal-networker/internal/netconf.applyAndCleanUp\n\tgithub.com/metal-stack/metal-networker/internal/netconf/configurator.go:223\ngithub.com/metal-stack/metal-networker/internal/netconf.FirewallConfigurator.Configure\n\tgithub.com/metal-stack/metal-networker/intern
al/netconf/configurator.go:101\ngithub.com/metal-stack/metal-networker/cmd.configure\n\tgithub.com/metal-stack/metal-networker/cmd/root.go:96\ngithub.com/metal-stack/metal-networker/cmd.glob..func1\n\tgithub.com/metal-stack/metal-networker/cmd/firewall.go:24\ngithub.com/spf13/cobra
.(*Command).execute\n\tgithub.com/spf13/cobra@v1.1.1/command.go:850\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\tgithub.com/spf13/cobra@v1.1.1/command.go:958\ngithub.com/spf13/cobra.(*Command).Execute\n\tgithub.com/spf13/cobra@v1.1.1/command.go:895\ngithub.com/metal-stack/metal-n
etworker/cmd.Execute\n\tgithub.com/metal-stack/metal-networker/cmd/root.go:43\nmain.main\n\tgithub.com/metal-stack/metal-networker/main.go:6\nruntime.main\n\truntime/proc.go:204"}                                                                                                       
panic: exit status 1: /etc/metal/networker/rules.v4_205597697:45:39-44: Error: Could not resolve hostname: Address family for hostname not supported                                                                                                                                      
        oifname "vlan104009" ip saddr 2002::/64 counter masquerade comment "snat (networkid: internet-vagrant-lab)"                                                                                                                                                                       
                                      ^^^^^^                                                                                                                                                                                                                                              
                                                                                                                                                                                                                                                                                          
                                                                                                                                                                                                                                                                                          
goroutine 1 [running]:                                                                                                                                                                                                                                                                    
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc000250180, 0x0, 0x0, 0x0)                                                                                                                                                                                                                
        go.uber.org/zap@v1.16.0/zapcore/entry.go:234 +0x585                                                                                                                                                                                                                               
go.uber.org/zap.(*SugaredLogger).log(0xc000011500, 0xc00008c204, 0x0, 0x0, 0xc0007155d8, 0x1, 0x1, 0x0, 0x0, 0x0)                                                                                                                                                                         
        go.uber.org/zap@v1.16.0/sugar.go:234 +0xf6                                                                                                                                                                                                                                        
go.uber.org/zap.(*SugaredLogger).Panic(...)                                                                                                                                                                                                                                               
        go.uber.org/zap@v1.16.0/sugar.go:123                                                                                                                                                                                                                                              
github.com/metal-stack/metal-networker/internal/netconf.mustApply(0xc050c0, 0xc000273b90, 0xc00011ea80, 0x92a, 0xc00008c2a0, 0x27, 0xb2ffca, 0x16)                                                                                                                                        
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:249 +0x1de                                                                                                                                                                                                
github.com/metal-stack/metal-networker/internal/netconf.applyAndCleanUp(0xc050c0, 0xc000273b90, 0xb2a587, 0xc, 0xc00008c2a0, 0x27, 0xb2ffca, 0x16, 0x180)                                                                                                                                 
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:223 +0x1cb                                                                                                                                                                                                
github.com/metal-stack/metal-networker/internal/netconf.FirewallConfigurator.Configure(0xc000244860, 0x3, 0x0, 0x0, 0x0, 0x0, 0xc000122600, 0x3, 0x3, 0xc00008c1b0, ...)                                                                                                                  
        github.com/metal-stack/metal-networker/internal/netconf/configurator.go:101 +0x1b9                                                                                                                                                                                                

```

**Comment by mwindower** [comment]
can be closed after #42 was merged
